src/app/satellite.ts

```diff
@@ -15,8 +15,11 @@ export class Satellite {
    }
 	
 	isSpaceDebris(): boolean {
-		return true;
-   }
+		if (this.type === "Space Debris") {
+			return true;
+		}
+		return false;
+   } 
```

Comment:
This is great, but can be simplified like so:

`return this.type === "Space Debris"`
This will evaluate the statement and return the result of the statement which will either be true or false